rls/src/test/java/io/grpc/rls/CachingRlsLbClientTest.java

```diff
@@ -172,6 +173,7 @@ private void setUpRlsLbClient() {
   public void tearDown() throws Exception {
     rlsLbClient.close();
     CachingRlsLbClient.enableOobChannelDirectPath = existingEnableOobChannelDirectPath;
+    assertThat(lbProvider.loadBalancers).isEmpty();
```

Comment:
Since this is in tearDown, maybe we should add a message? Something like

```java
assertWithMessage("On client shut down, RlsLoadBalancer must shut down with all its child loadbalancers.").that(lbProvider.loadBalancers).isEmpty();
```

Reply:
Great suggestion!